src/components/Transaction/Transaction.component.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
```

Comment:
Just noticed that we need to set `moment`'s locale. Since we don't use the library much (yet), a simple fix for now could be to do it at the start of the render method.
See http://momentjs.com/docs/#/i18n/

Reply:
![screenshot from 2018-10-16 15-07-13](https://user-images.githubusercontent.com/16665596/46992332-3b2c8980-d155-11e8-8cb6-c97e84636614.png)

Reply:
Great spot. Thanks for the note. Let me take a look.

Reply:
Made a fix to this. Please take a look. Thanks